src/pretix/base/models/items.py

```diff
@@ -369,10 +369,40 @@ def check_quotas(self, ignored_quotas=None, count_waitinglist=True, subevent=Non
         return min([q.availability(count_waitinglist=count_waitinglist, _cache=_cache) for q in check_quotas],
                    key=lambda s: (s[0], s[1] if s[1] is not None else sys.maxsize))
 
+    def allow_delete(self):
+        from pretix.base.models.orders import CartPosition, OrderPosition
+
+        return (
+            not OrderPosition.objects.filter(item=self).exists()
+            and not CartPosition.objects.filter(item=self).exists()
+        )
```

Comment:
We could also use this in ``pretix/control/views/item.py:1026``, don't you think?

Reply:
Done. However it seemed that check was only for order_positions, not for cart_positions?

Reply:
Yeah, but that was probably more of an oversight. I'm not even sure what the "right" approach is, since blocking because of CartPositions is confusing since the admin cannot see them anywhere. On the other hand, once the item is "out there" it is okay if you can't delete it since you can still disable it.